### DISCUSSION TITLE
How can I have a little bit of money

### DISCUSSION THREAD

**Discussion by VingeChen** [Chia-Network/chia-blockchain#2244] [General]
![image](https://user-images.githubusercontent.com/30143245/115505063-7c183280-a2ab-11eb-9ec7-02160f8c0087.png)

谁能给转一点点啊，新手不知道怎么玩
Chia 钱包：xch1vzh8qzqfqt0uu7y8aetpanrvr3eqzrc66cmdcqwgjr8dtu30st6q76aarw

**Comment by cyr616** [comment]
join hpool you can get about 0.06XCH per day now.
solo you probably get 2XCH in a mouth.

**Reply by VingeChen** [reply]
hpool  需要我的密钥呀

**Reply by ghost** [reply]
What is the difference between a Chia calculator and a hpool?
